include/RAJA/policy/openmp/policy.hpp

```diff
@@ -190,6 +194,7 @@ using policy::omp::omp_for_exec;
 using policy::omp::omp_for_nowait_exec;
 using policy::omp::omp_for_static;
 using policy::omp::omp_parallel_exec;
```

Comment:
I thought the idea was to use the omp_region (should probably be called `omp_parallel_region`) to implement some of the other `omp_parallel` policies?

Reply:
I was aiming for something  mentioned in #353 .

Reply:
Yup, and @trws suggested:
> One thing I’d say here, as a test of whatever you do, refactor the
existing internal omp parallel to use it.  That should give it a decent
sanity check for interactions with everything else.

Reply:
Oh I think know what you mean, internally have it do the following 
RAJA::region<omp_region>([=](){
    using RAJA::internal::thread_privatize;
    auto body = thread_privatize(loop_body);
    forall_impl(InnerPolicy{}, std::forward<Iterable>(iter), body.get_priv());
});

right?